cockroachcloud/managing-cmek.md

```diff
@@ -41,27 +43,26 @@ This section gives a high level overview of the operations involved with impleme
 
 1. To start the process, contact {{ site.data.products.dedicated }} by reaching out to your account team, or [creating a support ticket](https://support.cockroachlabs.com/). You must provide your **Organization ID**, which you can find in your [Organization settings page](https://cockroachlabs.cloud/settings).
 
-	They will enable the CMEK feature for your {{ site.data.products.dedicated }} Organization.
+    They will enable the CMEK feature for your {{ site.data.products.dedicated }} Organization.
 
 1. [Create a {{ site.data.products.db }} service account](console-access-management.html#service-accounts).
-
 1. [Create an API key](console-access-management.html#create-api-keys) for the service account to use.
 
 ### Step 2. Provision your cluster
 
-1. Create a new {{ site.data.products.dedicated }} cluster. There are two ways to do this:
-	- [Using the {{ site.data.products.db }} console clusters page](https://cockroachlabs.cloud/cluster).
-	- [Using the Cloud API](cloud-api.html#create-a-new-cluster). 
+Create a new {{ site.data.products.dedicated }} cluster. There are two ways to do this:
+
+- [Using the {{ site.data.products.db }} console clusters page](https://cockroachlabs.cloud/cluster).
+- [Using the Cloud API](cloud-api.html#create-a-new-cluster). 
 
 ### Step 3. Provision IAM and KMS in your Cloud
 
 Next, you must provision the resources required resources in your Cloud, whether this is AWS or GCP:
 
-1. The cross-account IAM role (in AWS) or service account (in GCP) that {{ site.data.products.dedicated }} will use to access your key.
-
 1. The key itself.
+1. The authorization object that grants the {{ site.data.products.db }} service account permission to encrypt and decrypt using the key. The authorization object is either an IAM role in AWS or a service account in GCP.
 
-Follow the instructions depending on your cloud provider:
+Follow the instructions for your {{ site.data.products.db }} Information-As-A-Service (IAAS) platform:
```

Comment:
Can I request to please keep it cloud provider, and not IaaS (across this whole CMEK doc)? I'm just not comfortable with the IaaS term all across our docs. AWS, GCP and Azure are not just IaaS providers, they've PaaS & SaaS services too. In fact, KMS is a SaaS service. I don't know who came up with this idea of referring to them as IaaS platforms, but this needs correction across our docs.